### ISSUE TITLE
MariaBD error

### ISSUE THREAD

**Issue by mdigger** [bokwoon95/sqddl#1]
> `tables: version 10.11.2-MariaDB: cannot convert 2-MariaDB to integer: strconv.Atoi: parsing "2-MariaDB": invalid syntax`

**Comment by mdigger** [comment]
```
dump: introspection_scripts/mysql_indexes.sql:
SELECT
    table_schema
    ,table_name
    ,index_name
    ,index_type
    ,is_unique
    ,group_concat(column_name ORDER BY seq_in_index) AS columns
    ,group_concat(is_descending ORDER BY seq_in_index) AS descending
FROM (
    SELECT
        table_schema
        ,table_name
        ,index_name
        ,index_type
        ,NOT non_unique AS is_unique
        ,COALESCE(column_name, CONCAT('(', expression, ')')) AS column_name
        ,CASE collation WHEN 'D' THEN 1 ELSE 0 END AS is_descending
        ,seq_in_index
    FROM
        information_schema.statistics
    WHERE
        NOT EXISTS (
            -- exclude indexes created by constraints
            SELECT 1
            FROM information_schema.table_constraints
            WHERE table_constraints.constraint_name = statistics.index_name
        )
        AND table_schema NOT IN ('mysql', 'information_schema', 'performance_schema', 'sys')
        AND table_name NOT IN ('sqddl_history')
    ) AS indexed_columns
GROUP BY
    table_schema
    ,table_name
    ,index_name
    ,index_type
    ,is_unique
ORDER BY
    table_schema
    ,table_name
    ,index_name
;
: Error 1054 (42S22): Unknown column 'expression' in 'field list'
```

**Comment by bokwoon95** [comment]
Aah.. I never tested for MariaDB and it looks like MariaDB doesn't have the 'expression' column in the information_schema.statistics table. I'll spin up a test MariaDB instance and see what else is incompatible.

**Comment by bokwoon95** [comment]
Seems like this is the only breaking change between MariaDB and MySQL as far as I can tell. Give the latest version a try (v0.4.0) and see if anything else breaks for you.

**Comment by bokwoon95** [comment]
Resolved by [600b8fc](https://github.com/bokwoon95/sqddl/commit/600b8fc851e6ecd425b0102e5826cbead4c54ad8)
